core-services/image-mirroring/openshift/mapping_origin_4_11

```diff
@@ -46,7 +46,10 @@ registry.ci.openshift.org/origin/4.11:cluster-autoscaler-operator quay.io/opensh
 registry.ci.openshift.org/origin/4.11:cluster-baremetal-operator quay.io/openshift/origin-cluster-baremetal-operator:4.11 quay.io/openshift/origin-cluster-baremetal-operator:4.11.0 quay.io/openshift/origin-cluster-baremetal-operator:latest
 registry.ci.openshift.org/origin/4.11:cluster-bootstrap quay.io/openshift/origin-cluster-bootstrap:4.11 quay.io/openshift/origin-cluster-bootstrap:4.11.0 quay.io/openshift/origin-cluster-bootstrap:latest
 registry.ci.openshift.org/origin/4.11:cluster-capacity quay.io/openshift/origin-cluster-capacity:4.11 quay.io/openshift/origin-cluster-capacity:4.11.0 quay.io/openshift/origin-cluster-capacity:latest
+registry.ci.openshift.org/origin/4.11:cluster-capi-aws-controllers quay.io/openshift/origin-cluster-capi-aws-controllers:4.11 quay.io/openshift/origin-cluster-capi-aws-controllers:4.11.0 quay.io/openshift/origin-cluster-capi-aws-controllers:latest
+registry.ci.openshift.org/origin/4.11:cluster-capi-azure-controllers quay.io/openshift/origin-cluster-capi-azure-controllers:4.11 quay.io/openshift/origin-cluster-capi-azure-controllers:4.11.0 quay.io/openshift/origin-cluster-capi-azure-controllers:latest
 registry.ci.openshift.org/origin/4.11:cluster-capi-controllers quay.io/openshift/origin-cluster-capi-controllers:4.11 quay.io/openshift/origin-cluster-capi-controllers:4.11.0 quay.io/openshift/origin-cluster-capi-controllers:latest
+registry.ci.openshift.org/origin/4.11:cluster-capi-gcp-controllers quay.io/openshift/origin-cluster-capi-gcp-controllers:4.11 quay.io/openshift/origin-cluster-capi-gcp-controllers:4.11.0 quay.io/openshift/origin-cluster-capi-gcp-controllers:latest
```

Comment:
see https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/GUIDELINES.md

admittedly there are two conflicting guidelines there, in this case, but i think we want the platform prefix to come first, so `gcp-capi-controllers` (i don't think we need "cluster" in the name) and similar for the other ones.

@smarterclayton agree?

Reply:
yeah, `<platform>-<clear_name_describing_function>-<type_of_image>`.

I don't think CAPI is sufficient on its own, i would probably say `gcp-cluster-api-controllers` is the pattern I'd expect

Reply:
thanks, all fixed. My intention was to have all CAPI related images near each other in the list.

Reply:
Understood, we tilted this way because ultimately we want to be clear about what components are required on a given cluster, and like other cloud provider resources there can be only one active at a time.